Ambit/Source/Ambit/Mode/AmbitWidget.cpp

```diff
@@ -78,18 +78,7 @@ void SAmbitWidget::Construct(const FArguments& InArgs, TSharedRef<FAmbitModeTool
 
     FAmbitMode* AmbitMode = GetEditorMode();
 
-    // Register the Ambit tutorial category with the tutorial browser.
-    FTutorialCategory AWSAmbitCategory = FTutorialCategory();
-    AWSAmbitCategory.Identifier = "AWSAmbit";
-    AWSAmbitCategory.Title = NSLOCTEXT("TutorialCategories", "AWSAmbitTitle", "AWS Ambit");
-    AWSAmbitCategory.Description = NSLOCTEXT("TutorialCategories", "AWSAmbitDescription", "This tutorial will cover how AWS Ambit works.");
-    AWSAmbitCategory.Icon = "LevelEditor.Ambit";
-    AWSAmbitCategory.Texture = FSoftObjectPath("/Ambit/Icons/AmbitIcon");
-    AWSAmbitCategory.SortOrder = 100;
-
     IIntroTutorials& IntroTutorials = FModuleManager::LoadModuleChecked<IIntroTutorials>(TEXT("IntroTutorials"));
```

Comment:
Why is this statement not deleted?

Reply:
This line was used for the following code in this file to create that tutorial button.